src/components/MessagesList/MessagesList.vue

```diff
@@ -653,70 +655,27 @@ export default {
 				})
 
 				if (this.$store.getters.getFirstKnownMessageId(this.token) === null) {
-					let startingMessageId = 0
-					// first time load, initialize important properties
-					if (focusMessageId === null) {
-						// Start from unread marker
-						this.$store.dispatch('setFirstKnownMessageId', {
-							token: this.token,
-							id: this.conversation.lastReadMessage,
-						})
-						startingMessageId = this.conversation.lastReadMessage
-						this.$store.dispatch('setLastKnownMessageId', {
-							token: this.token,
-							id: this.conversation.lastReadMessage,
-						})
-					} else {
-						// Start from message hash
-						this.$store.dispatch('setFirstKnownMessageId', {
-							token: this.token,
-							id: focusMessageId,
-						})
-						startingMessageId = focusMessageId
-						this.$store.dispatch('setLastKnownMessageId', {
-							token: this.token,
-							id: focusMessageId,
-						})
-					}
+					// Start from message hash or unread marker
+					const startingMessageId = focusMessageId !== null ? focusMessageId : this.conversation.lastReadMessage
+					// First time load, initialize important properties
+					this.$store.dispatch('setFirstKnownMessageId', { token: this.token, id: startingMessageId })
+					this.$store.dispatch('setLastKnownMessageId', { token: this.token, id: startingMessageId })
 
 					// Get chat messages before last read message and after it
 					await this.getMessageContext(startingMessageId)
-					const startingMessageFound = this.focusMessage(startingMessageId, false, focusMessageId !== null)
-
-					if (!startingMessageFound) {
-						const fallbackStartingMessageId = this.$store.getters.getFirstDisplayableMessageIdBeforeReadMarker(this.token, startingMessageId)
-						this.$store.dispatch('setVisualLastReadMessageId', {
-							token: this.token,
-							id: fallbackStartingMessageId,
-						})
-						this.focusMessage(fallbackStartingMessageId, false, false)
-					}
 				}
 
-				let hasScrolled = false
-				if (focusMessageId === null) {
-					// if lookForNewMessages will long poll instead of returning existing messages,
-					// scroll right away to avoid delays
-					if (!this.hasMoreMessagesToLoad) {
-						hasScrolled = true
-						this.$nextTick(() => {
-							this.scrollToFocusedMessage(focusMessageId)
-						})
-					}
-				}
+				this.$nextTick(() => {
+					// basically scrolling to either the last read message or the message in the URL anchor
+					// and there is a fallback to scroll to the bottom if the message is not found
+					this.scrollToFocusedMessage(focusMessageId)
+				})
```

Comment:
Should prevent unintentional scrollToBottom
```suggestion
					this.isInitialisingMessages = false
				})
```